infra/modules/service/waf.tf

```diff
@@ -0,0 +1,216 @@
+resource "aws_wafv2_web_acl" "waf" {
+  name  = "wafv2-web-acl"
+  scope = "REGIONAL"
+
+  default_action {
+    allow {
```

Comment:
A nit, but lets go through and either have all of these curly braces open, or closed. I prefer closed for empty blocks `allow {}`, `none {}` etc, but I am open to viewpoint.